### PR TITLE
Fix mutil-match crash

### PR DESCRIPTION
**Pull request by czpmango**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 
https://github.com/vesoft-inc/nebula-ent/issues/1489

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [x] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by jievince — APPROVED** [review]

File: src/graph/planner/plan/PlanNode.cpp [hunk 1]
```diff
@@ -423,7 +423,11 @@ BinaryInputNode::BinaryInputNode(QueryContext* qctx,
   }
 
   addDep(right);
-  readVariable(right->outputVarPtr());
+  if (right != nullptr) {
```
Comment (Shylock-Hg) [inline src/graph/planner/plan/PlanNode.cpp:426]: Why right will be null?

**Review by jievince — APPROVED** [review]

File: src/graph/planner/plan/PlanNode.cpp (hunk 1 above)
Comment (czpmango) [reply src/graph/planner/plan/PlanNode.cpp:426]: Just like BinaryInputNode(qctx, kind, nullptr, nullptr)?

Comment (czpmango) [reply src/graph/planner/plan/PlanNode.cpp:426]: BiLeftJoin::clone()

**Review by codesigner — APPROVED** [review]
LGTM

**Review by Shylock-Hg — APPROVED** [review]
LGTM, maybe need a constructor for clone.

**Pull request merged**